analyzers/tests/SonarAnalyzer.Test/Rules/AspNet/RouteTemplateShouldNotStartWithSlashTest.cs

```diff
@@ -26,8 +26,8 @@ namespace SonarAnalyzer.Test.Rules;
 [TestClass]
 public class RouteTemplateShouldNotStartWithSlashTest
 {
-    private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.RouteTemplateShouldNotStartWithSlash>();
-    private readonly VerifierBuilder builderVB = new VerifierBuilder<VB.RouteTemplateShouldNotStartWithSlash>();
+    private readonly VerifierBuilder builderCS = new VerifierBuilder<CS.RouteTemplateShouldNotStartWithSlash>().WithBasePath("AspNet");
+    private readonly VerifierBuilder builderVB = new VerifierBuilder<VB.RouteTemplateShouldNotStartWithSlash>().WithBasePath("AspNet");
```

Comment:
I wonder whether it would be better to create a constant, since the value is used many times across multiple source files.

If we want the constant, it should be created in a helper at common level such as `SonarAnalyzer.TestFramework/Common/Constants` or somewhere in `SonarAnalyzer.Test` (which seems more correct, since the constant would relate to the folder structure of the `SonarAnalyzer.Test` project, and has nothing to do with the Test Framework as such).

At the same time, we don't seem to use such constants for `SymbolicExecution/Roslyn`, `SymbolicExecution/Sonar`, `Hotspots` etc. We always replicate the string, so there may be good reasons to do so, and you may want to ask the team for other opinions.

The constant would come in handy if we change the folder name, as we would not need to do search and replace of `.WithBasePath("AspNet")`, but the advantages end there, since, once you have the static using, it's only 2 chars longer to write `.WithBasePath("AspNet")` than `.WithBasePath(AspNet)`, and equally expressive.

This remark shouldn't be blocking the PR, since we use string literals instead of constants in all other cases (symbolic execution, cloud native, hotspots, etc.). If, after checking with the squad, we want to make the change, in my opinion it should be done in a dedicated PR.

Reply:
I'm really not sure :/ 
If we change the path the tests will fail - thus easy to catch.

I'll apply this in the next PR for S6962 and see how it looks.